File: app/src/main/java/work/beltran/discogsbrowser/api/model/RecordsWithPagination.java
package work.beltran.discogsbrowser.api.model;

import java.util.List;

import work.beltran.discogsbrowser.api.model.pagination.Pagination;
import work.beltran.discogsbrowser.api.model.record.Record;

/**
 * Created by Miquel Beltran on 04.05.16.
 * More on http://beltran.work
 */
public interface RecordsWithPagination {
    Pagination getPagination();
    List<Record> getRecords();
}

File: data-api/src/main/java/work/beltran/discogsbrowser/api/model/RecordsWithPagination.kt
package work.beltran.discogsbrowser.api.model

import work.beltran.discogsbrowser.api.model.pagination.Pagination
import work.beltran.discogsbrowser.api.model.record.RecordApi

/**
 * Created by Miquel Beltran on 04.05.16.
 * More on http://beltran.work
 */
interface RecordsWithPagination {
    val pagination: Pagination
    val records: List<RecordApi>
}
